client/components/jetpack-logo/index.jsx

```diff
@@ -10,10 +10,14 @@ import PropTypes from 'prop-types';
  * Module constants
  */
 const logoPathSize32 = (
-	<path
-		className="jetpack-logo__icon"
-		d="M16,0C7.2,0,0,7.2,0,16s7.2,16,16,16c8.8,0,16-7.2,16-16S24.8,0,16,0z M15.2,18.7h-8l8-15.5V18.7z M16.8,28.8 V13.3h8L16.8,28.8z"
-	/> // eslint-disable-line max-len
+	<g>
+		<path
+			className="jetpack-logo__icon-circle"
+			d="M16,0C7.2,0,0,7.2,0,16s7.2,16,16,16s16-7.2,16-16S24.8,0,16,0z"
+		/>
+		<polygon className="jetpack-logo__icon-triangle" fill="#fff" points="15,19 7,19 15,3 " />
+		<polygon className="jetpack-logo__icon-triangle" fill="#fff" points="17,29 17,13 25,13 " />
+	</g>
 );
 const svgLogo24 = (
```

Comment:
Intentional no white?

Reply:
Yeah. The 24px version is more like a gridicon where it should follow gridicon styles.